{{cookiecutter.project_directory_name}}/Makefile

```diff
@@ -0,0 +1,15 @@
+#################################################################################
+# COMMANDS                                                                      #
+#################################################################################
+
+venv:
```

Comment:
should there be an R-equivalent of virtual envs ?

Reply:
This will be addressed in the next pull request.